### DISCUSSION TITLE
Talos crashes on Proxmox server before getting to maintenance mode

### DISCUSSION THREAD

**Discussion by alexisHouss** [siderolabs/talos#6313] [Q&A] [answered]
Hi,

I am fairly new using Proxmox and Talos. I have set my proxmox server on a dedicated server with only one ipv4 and created a second Linux Bridge (vmbr1) for private network as well as DHCP 'server' container (Ubuntu 18.04 with dnsmasq running). The DHCP server works as I have created a VM with Debian to check and it gets an ip assigned.
When creating a VM with Talos, it crashes whilst loading the maintenance mode. I have tried to set up the ip myself from the 'without DHCP server' documentation part (https://www.talos.dev/v1.2/talos-guides/install/virtualized-platforms/proxmox/#without-dhcp-server).
However, it seems that nothing works. Not sure what to change.

What are the best practices when setting up a VM with Talos os on a Proxmox server?

**Comment by frezbo** [comment, marked as answer]
You'd have to set the cpu mode to host or expose the cpu micro architectures. This is mentioned in the https://www.talos.dev/v1.2/talos-guides/install/virtualized-platforms/proxmox/#create-vms

**Reply by alexisHouss** [reply]
Thank you very much! It brings another error but at least it doesn't crash anymore :).
